doc/rbd/iscsi-target-cli.rst

```diff
@@ -0,0 +1,166 @@
+=============================================================
+Configuring the iSCSI Target using the Command Line Interface
+=============================================================
+
+The Ceph iSCSI gateway is the iSCSI target node and also a Ceph client
+node. The Ceph iSCSI gateway can be a standalone node or be colocated on
+a Ceph Object Store Disk (OSD) node. Completing the following steps will
+install, and configure the Ceph iSCSI gateway for basic operation.
+
+**Requirements:**
+
+-  A running Ceph Luminous or later storage cluster
+
+-  CentOS 7.4, Linux kernel v4.12 or newer
```

Comment:
This will now require a 4.14 kernel. Note that this will not be released for several months.

You could use 4.13, but the user would have to manually patch the kernel with these patches:
http://www.spinics.net/lists/target-devel/msg15942.html
I do not think we want to document how to apply patches, so maybe it is easier to just write that users should use 4.14 when it comes out?